core/src/main/java/bisq/core/payment/payload/PaymentAccountPayload.java

```diff
@@ -70,6 +73,11 @@
     @Nullable
     protected final Map<String, String> excludeFromJsonDataMap;
 
+    @JsonExclude
+    @Getter(AccessLevel.NONE)
+    @Setter    
+	protected String paymentDetails;
+
```

Comment:
Please don't use the protocol buffers files to store this kind of information. It is very risky and it needs additional backwards comp. testing to make sure everything is 100% working.

Reply:
I was not talking about the getPaymentDetails method I was talking about the property `paymentDetails` you have added in this PR.

Reply:
OK. Apologies for the mix-up.